src/mapeo-project.js

```diff
@@ -124,4 +166,34 @@ export class MapeoProject {
   get field() {
     return this.#dataTypes.field
   }
+
+  /**
+   * @param {Omit<import('@mapeo/schema').ProjectValue, 'schemaName'>} settings
```

Comment:
kind of forgot what `defaultPresets` represents - should it also be omitted here?

Reply:
Is it possible to modify the schema given https://github.com/digidem/mapeo-schema/issues/104?

Reply:
Yes, we allow fields to be `undefined`. We just don't allow them to be `null`

Reply:
issue created: https://github.com/digidem/mapeo-schema/issues/132

Reply:
> I think this should be Partial<Omit<ProjectValue, 'schemaName'>>

addressed via https://github.com/digidem/mapeo-core-next/pull/187/commits/2cc9d44ba14b587b5a4f52b118d6d2cd11ad29a5